01_prepare_host.sh

```diff
@@ -79,6 +81,12 @@ if ! command -v kustomize 2>/dev/null ; then
     rm "kustomize_${KUSTOMIZE_VERSION}_linux_amd64.tar.gz"
 fi
 
+if ! [ -x "$(command -v clusterctl)" ]; then
+  curl -L https://github.com/kubernetes-sigs/cluster-api/releases/download/"${CAPIRELEASE}"/clusterctl-linux-amd64 -o clusterctl
+  chmod +x ./clusterctl
+  sudo mv ./clusterctl /usr/local/bin/clusterctl
+fi
```

Comment:
what if CAPIRELEASE changes in between two runs ? you should probably check the version and if it is not in sync with the release version, upgrade.

Reply:
Fixed, thanks.